backend/src/main/java/com/google/sps/data/Match.java

```diff
@@ -11,9 +11,9 @@
   /** Datastore entity ID */
   private final long id;
   /** First participant */
-  private final Participant firstParticipant;
+  private final com.google.sps.data.Participant firstParticipant;
```

Comment:
`import com.google.sps.data.Participant` so we don't need the full name all the time?

Reply:
I cannot import the Class due to the how the packages are defined causing the import to go unused

Reply:
If this will compile as-is then import com.google.sps.data.Participant should work too

Reply:
Whenever i import com.google.sps.data.Participant the linter fails due to unused import staements

Reply:
Ah, I see why. Participant is also in this package so it can be referenced just as `Participant` without an import statement.

Reply:
Acknowledged